xcode/Subconscious/Shared/Library/NaturalLanguage/TraceryDisambiguator.swift

```diff
@@ -12,57 +12,106 @@ struct TraceryContext: Hashable {
     var grammar: Grammar = [:]
 }
 
+enum DisambiguatorTag {
+    case journal
+    case project
+    case list
+    case link
+    case quote
+    case heading
+}
```

Comment:
I can imagine we might expand this to include other generally-useful information to expose to routes, such as tokenized/lemmatized words, nouns, etc.